<launch>
  <!-- ========================= -->
  <!-- (1) Map Server: 맵 퍼블리시 -->
  <!-- ========================= -->
  <node name="map_server" pkg="map_server" type="map_server"
        args="$(find kora_k3)/maps/f3.yaml"
        output="screen" />

  <!-- ========================= -->
  <!-- (3) Pure Pursuit 관련 노드 -->
  <!-- ========================= -->
  <!-- <node pkg="pure_pursuit" type="pure_pursuit.py"
        name="pure_pursuit_node" output="screen" /> -->

  <node pkg="pure_pursuit" type="waypoint_visualization.py"
        name="waypoint_visualization_node" output="screen" />

</launch>
